bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/Clip2Bridge.java

```diff
@@ -1106,17 +1107,17 @@ public void putResource(Resource resource) throws ApiException, InterruptedExcep
             String contentType = contentStreamListener.getContentType();
             int status = contentStreamListener.getStatus();
             LOGGER.trace("HTTP/2 {} (Content-Type: {}) << {}", status, contentType, contentJson);
-            if (status != HttpStatus.OK_200) {
+            if (!HttpStatus.isSuccess(status)) {
```

Comment:
As I don’t have access to a PC can you please confirm that isSuccess() returns true for all 20x errors?

Reply:
It does exactly that, here is the implementation:
```java
    public static boolean isSuccess(int code)
    {
        return ((200 <= code) && (code <= 299));
    }
```

Reply:
Ok. Thanks.